src/app/zap-templates/templates/app/helper.js

```diff
@@ -178,6 +180,30 @@ function chip_endpoint_generated_functions()
   return ret.concat('\n');
 }
 
+function chip_endpoint_generated_commands_list()
+{
+  let ret = '{';
+  this.clusterList.forEach((c) => {
+    clientGeneratedCommands = [];
+    serverGeneratedCommands = [];
+    c.commands.forEach((cmd) => {
+      console.log(cmd);
```

Comment:
We probably don't want to check the log statement here in.

Reply:
Removed debug log.